distribution/scripts/jmeter/run-performance-tests.sh

```diff
@@ -83,10 +83,10 @@ declare -A test_scenario3=(
     [skip]=false
 )
 declare -A test_scenario4=(
-    [name]="passthrough_http2_https"
-    [display_name]="Passthrough HTTP2 (HTTPS) service"
-    [bal]="http2_https_passthrough.balx"
-    [description]="An HTTPS Service exposed over HTTP2 protocol, which forwards all requests to a HTTP2 (HTTPS) back-end service."
+    [name]="h2_h2_passthrough"
+    [display_name]="Passthrough HTTP2 (HTTPS) service (h2 -> h2)"
```

Comment:
Feels like this display name is misleading. For HTTP/1.1 over TLS scenario its called HTTPS  and here also in brackets, it says HTTPS which gives a wrong meaning. I would rather use "over TLS" inside brackets.

Reply:
Agreed. Fixed with https://github.com/ballerina-platform/ballerina-performance/pull/23/commits/0b4d53cff493f87629996209363f02f91919c66d